pages/api/users/index.ts

```diff
@@ -0,0 +1,120 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { CreateUser, User } from '~/shared/types';
+import prisma from '~/prisma';
+import * as bcrypt from 'bcrypt';
```

Comment:
Näitä `import * as` kannattaa välttää jos mahdollista. Tässä parempi importata vain hash mikä tarvitaan.
```suggestion
import { hash } from 'bcrypt';
```